documentation/migration-guide.md

```diff
@@ -0,0 +1,200 @@
+> `rdme@10` has not been released yet. If you're using ReadMe Refactored, stay tuned!
+<!-- prettier-ignore-end -->
+
+### Overview
+
+A [bi-directional syncing](https://docs.readme.com/main/docs/bi-directional-sync) workflow with [ReadMe Refactored](https://docs.readme.com/main/docs/welcome-to-readme-refactored) mostly eliminates the need for a tool like `rdme`. For syncing Markdown files, syncing API definitions, and managing project hierarchy (e.g., project versions and categories) with ReadMe Refactored, you'll want to set up bi-directional syncing.
+
+`rdme@10` is recommended for the following use cases:
+
+- Syncing your API definition (generated via a build process and not tracked via Git) to your ReadMe Refactored-enabled project
+- Syncing Markdown files to the Changelog for your ReadMe Refactored-enabled project
+
+<!-- prettier-ignore-start -->
+> [!NOTE]
+> `rdme@10` only works with ReadMe projects that are using [ReadMe Refactored](https://docs.readme.com/main/docs/welcome-to-readme-refactored). If you are not yet using ReadMe Refactored, [you'll want to use `rdme@9`](#migrating-to-rdme9).
+<!-- prettier-ignore-end -->
+
+### Upgrading to `v10`
+
+#### Step 1: Upgrade via `npm`
+
+To install this version of the `rdme` CLI globally, run the following command:
+
+```sh
+npm install -g rdme@10
+```
+
+More installation options can be found in [our docs](https://github.com/readmeio/rdme/tree/v10?tab=readme-ov-file#setup).
+
+#### Step 2: Update GitHub Actions Workflow
+
+If you're using the `rdme` GitHub Action, update your GitHub Actions workflow file so your `rdme` usage uses the `v10` reference like so:
+
+```yaml
+- uses: readmeio/rdme@v10
+  with:
+    rdme: openapi validate petstore.json
+```
+
+#### Step 3: Address Breaking Changes
+
+1. **Enable Bi-Directional Syncing** _(recommended)_
+
+   We recommend setting up [bi-directional syncing](https://docs.readme.com/main/docs/bi-directional-sync) for managing your Markdown files, API definitions and project hierarchy.
+
+2. **Command Replacements**
+
+   - Replace: `categories` → use [Git-based workflow](https://docs.readme.com/main/docs/bi-directional-sync)
+   - Replace: `custompages` → use [Git-based workflow](https://docs.readme.com/main/docs/bi-directional-sync)
+   - Replace: `docs` (and its `guides` alias) → use [Git-based workflow](https://docs.readme.com/main/docs/bi-directional-sync)
+   - Replace: `versions` → use [Git-based workflow](https://docs.readme.com/main/docs/bi-directional-sync)
+   - Remove: `open`
+
+## Migrating to `rdme@9`
+
+### Overview
+
+This release adds a few features that make it even easier to get started with `rdme`:
+
+1. **Enhanced Command Documentation**
+
+   - Complete command reference in [the `documentation/commands` directory](https://github.com/readmeio/rdme/tree/v9/documentation/commands)
+   - Detailed usage examples and parameter descriptions
+   - Structured by command category for intuitive navigation
+
+2. **Improved CLI Experience**
```

Comment:
the other ones have a line break after the number
```suggestion
2. **Improved CLI Experience**

```

Reply:
ah nice catch. prettier auto-adds those line breaks everywhere else for me, no idea why it didn't do that here